runtimes/alibi-detect/mlserver_alibi_detect/cd/runtime.py

```diff
@@ -0,0 +1,33 @@
+from mlserver_alibi_detect.runtime import AlibiDetectRuntime
+from mlserver_alibi_detect.errors import InvalidAlibiDetector
+from mlserver.utils import get_model_uri
+from alibi_detect.utils.saving import load_detector
+import pickle
+
+
+class AlibiDriftDetectRuntime(AlibiDetectRuntime):
+    """
+    Implementation of the MLModel interface to load and serve Alibi drift models.
+    """
+
+    async def load(self) -> bool:
+
+        model_uri = await get_model_uri(self._settings)
+
+        if not self.alibi_detect_settings.init_detector:
+            self._model = load_detector(model_uri)
+        else:
+            try:
+                detector_data = pickle.load(open(model_uri, "rb"))
+                drift_detector = self.alibi_detect_settings.detector_type
+            except Exception as e:
```

Comment:
Should we catch such a generic exception? Is it just exceptions from `pickle` that we care about here?

Reply:
Yeah good point. I think I'll update it to (FileNotFoundError, EOFError, pickle.UnpicklingError) exceptions. I guess it covers the errors we are interested in here. Thoughts @jklaise ?

Reply:
Hmm guess that would cover the obvious cases, not an expert tho on catching errors when unpickling!

Reply:
Okay I have added these more specific exceptions now.